plugin/v2/healthz_test.go

```diff
@@ -0,0 +1,165 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v2
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"net/url"
+	"strconv"
+	"testing"
+	"time"
+
+	"google.golang.org/api/cloudkms/v1"
+	"google.golang.org/api/googleapi"
+
+	"github.com/phayes/freeport"
+
+	"github.com/GoogleCloudPlatform/k8s-cloudkms-plugin/plugin"
+)
+
+func TestHealthzServer(t *testing.T) {
+	t.Parallel()
+
+	var (
+		//positiveTestIAMResponse = &cloudkms.TestIamPermissionsResponse{
+		//	Permissions: []string{
+		//		"cloudkms.cryptoKeyVersions.useToDecrypt",
+		//		"cloudkms.cryptoKeyVersions.useToEncrypt",
+		//	},
+		//	ServerResponse: googleapi.ServerResponse{
+		//		HTTPStatusCode: http.StatusOK,
+		//	},
+		//}
+		negativeTestIAMResponse = &cloudkms.TestIamPermissionsResponse{
+			Permissions: []string{},
+			ServerResponse: googleapi.ServerResponse{
+				HTTPStatusCode: http.StatusOK,
+			},
+		}
+	)
+
+	testCases := []struct {
+		desc      string
+		query     string
+		response  []json.Marshaler
+		want      int
+		keySuffix string
+	}{
+		//{
```

Comment:
Why is this commented out?

Reply:
Sorry, I don't know. The test is left in tact as it was originally before the PR.